cronJob/cronJob.js

```diff
@@ -1,58 +1,56 @@
 const cron = require('node-cron');
-const subscriptionController = require('../controllers/subscriptionController');
 const request = require('request');
 const moment = require("moment");
-const {format} = require("winston");
 const {Subscription} = require('../models/subscriptions')
+const serverLogger = require(`../logger`);
+const sendQueueUpdate = require("../RMQ/rabbitMQ");
 
-require("dotenv").config({path: '.env'});
+const logger = serverLogger.log;
 
-const addMonthForDate =(subscription) => {
+
+const updateSubscription = (subscription, credits) => {
 
     let nextMonth = moment().add(1, 'M').format('YYYY-MM-DD HH:mm:ss');
     let updatedSubscription = new Subscription;
     updatedSubscription = subscription;
     updatedSubscription.next_date = nextMonth;
-    console.log("updatedSubscription:", updatedSubscription)
 
-    Subscription.findByIdAndUpdate(subscription._id, updatedSubscription, {new: true}, (err, docs) => {
+    Subscription.findByIdAndUpdate(subscription._id, updatedSubscription, {new: true}, (err) => {
         if (err) {
-            console.log(err)
+            logger.error(`findByIdAndUpdate failed: ${err} to user email: ${subscription.email}`);
+        } else {
+            logger.info(`Next date subscription updated successfully to user email: ${subscription.email}`);
+
+            sendQueueUpdate(subscription.email, credits);
+
         }
     })
 }
 
 
-
-
-
 const startCronJob = () => {
     let subscriptions;
     //code for 02 at night -cron.schedule("00 02 * * *", ()
 
-    cron.schedule("*/10 * * * * *", () => {
+    cron.schedule("*/6 * * * * *", () => {
         //get all subscriptions
-        request('http://localhost:'+process.env.PORT +'/subscription', {json: true}, (err, res, body) => {
+        request('http://localhost:' + process.env.PORT + '/subscription/getAllSubscriptionByName/Free', {json: true}, (err, res, body) => {
```

Comment:
remember that the host name should be a dynamic value. please move to config/ env var